packages/client/hmi-client/src/components/model/petrinet/tera-parameter-other-value-modal.vue

```diff
@@ -142,17 +151,37 @@ import Column from 'primevue/column';
 import Row from 'primevue/row';
 import ColumnGroup from 'primevue/columngroup';
 import RadioButton from 'primevue/radiobutton';
+import TeraLineGraphic from './tera-line-graphic.vue';
 
 const props = defineProps<{
-	id: string;
+	id: any;
 	otherValueList: any;
 }>();
 
-const otherValueList = ref(props.otherValueList);
+const otherValueList = ref(
+	props.otherValueList.map((value, index) => {
+		value.index = index + 1;
+		return value;
+	})
+);
+
+const minValues: Array<number> = [];
+const maxValues: Array<number> = [];
+otherValueList.value.forEach((element) => {
+	if (element.distribution?.parameters?.maximum) {
+		maxValues.push(element.distribution?.parameters?.maximum);
+	}
+	if (element.distribution?.parameters?.minimum) {
+		minValues.push(element.distribution?.parameters?.minimum);
+	}
+});
+const minValue: number = min(minValues) ?? 0;
+const maxValue: number = max(maxValues) ?? 0;
```

Comment:
This part is a bit faulty from a technical/completness sense, because you have a predicate conditioned on max/min, but they are numbers and can very well be 0 and disrupt your truthy checks

So for example if my maximums were `-2, -1, 0` the true max should be `0` but I think your algorithm will return me `-1`.

Also if you are sticking to D3, it has a method called `extent` where it figures out the max/min on your behalf, and you don't need to worry too much of what the values are in the process:

```
> d3.extent([1, 2, null, 4, 5, "blue", "green"])

< (2) [1, 5]
```